### PR TITLE
Blog Post: Additional Funding

### PR DESCRIPTION
**Pull request by dergigi**
In order:

- [x] Get approval from Donor
- [x] Finalize PR
- [x] Final sanity check by @OpenSats/volunteers 

**Build preview:**
- [/blog/opensats-receives-additional-funding...](https://os-website-git-startsmall2-open-sats.vercel.app/blog/opensats-receives-additional-funding-of-dollar21m-from-startsmall)

**Social preview:**
- 32-startsmall

---

To be published on **May 4th**

### CODE REVIEW

File: data/blog/opensats-receives-additional-funding-of-dollar21m-from-startsmall.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+><cite>—[jack][JD]</cite>
+
+We agree, and are honored and humbled to receive such a generous donation. We
+will do our best to continually allocate the funds to high-impact projects and
+[freedom tech infrastructure](/blog/bitcoin-infrastructure-this-decade).  As of
+this writing, OpenSats has allocated close to [\$10,000,000 in grants][EOY2023],
+supporting over 100 grantees, many of whom for the [long term](/tags/lts).
+
+As a 501(c)(3) public charity, we rely on donations from the general public,
+including philanthropic organizations such as #startsmall and the [Reynolds
```
Comment (Bayernatoor) [inline data/blog/opensats-receives-additional-funding-of-dollar21m-from-startsmall.mdx:29]:
```suggestion
including philanthropic organizations such as #startsmall and the [Reynolds
```
including philanthropic organizations such as [#startsmall](https://startsmall.llc/) and the [Reynolds

Comment (Bayernatoor) [reply data/blog/opensats-receives-additional-funding-of-dollar21m-from-startsmall.mdx:29]: Might want to link to #startsmall again?

File: data/blog/opensats-receives-additional-funding-of-dollar21m-from-startsmall.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+draft: false
+summary: "OpenSats receives $21 million from #startsmall for bitcoin, nostr, and related FOSS development."
+---
+
+
+We are thrilled to announce that OpenSats has received an additional donation of
+\$21M from [#startsmall](https://startsmall.llc/), Jack Dorsey's philanthropic
+initiative. This donation will top up our funds and operations budget, with
+\$15,000,000 going to our [General Fund][gf], \$5,000,000 to [The Nostr
+Fund][nf], and \$1M to our [operations budget][ops].
```
Comment (Bayernatoor) [inline data/blog/opensats-receives-additional-funding-of-dollar21m-from-startsmall.mdx:15]:
```suggestion
Fund][nf], and \$1,000,000 to our [operations budget][ops].
```
Change for consistency?

**Review by Bayernatoor — COMMENTED** [review]
2 minor cosmetic suggestions. 

Asides from it's good to go. Amazing news, big win for OpenSats and the many developers that this will help.

**Comment by btcschellingpt** [comment]
Monster announcement. Good to see the allocation which I agree is really prudent (Ops). Reads well and all links work. Hadn't reviewed #startsmall before .. fuck me dead, Jack is a goddamn philanthropic firehose. Bless him.

Good to go - make it so!

File: data/blog/opensats-receives-additional-funding-of-dollar21m-from-startsmall.mdx (hunk 1 above)
Comment (btcschellingpt) [reply data/blog/opensats-receives-additional-funding-of-dollar21m-from-startsmall.mdx:29]: +1

**Comment by abitcoinperson** [comment]
Maybe I'm reading this weird, but this sounds a bit strange image

**Pull request merged**